src/packages/examples-ext/processors/exampleDependenciesBuilder.js

```diff
@@ -36,15 +39,25 @@ module.exports = function exampleDependenciesBuilder (readFilesProcessor, log, g
 
       // traverse deployments, check project dependencies and add local file
       // dependencies to the list of generated documents
-      generateExamplesProcessor.deployments.forEach(function(deployment) {
```

Comment:
I experimented with your method but it doesn't actually alter the commonFiles passed to the deployments to add the iframeResizer. It is only added to the docs list but the commonFiles object is not modified.

As for code clarity... personally, I think using map/reduce is less clear than a for loop, but to each his own.